l10n_be_mis_reports/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```

Comment:
Can you make the module title a real title by adding some "===" above the title (https://github.com/OCA/maintainer-tools/pull/109)?